main.go

```diff
@@ -6,60 +6,126 @@ import (
 	"flag"
 	"fmt"
 	"io/fs"
+	"log"
 	"net"
 	"os"
 	"path/filepath"
 	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	ItemTypeTextFile  = 0
+	ItemTypeDirectory = 1
+	// TODO: add more
 )
```

Comment:
just convention, consts at top of file, usually it goes consts, vars, types, funcs